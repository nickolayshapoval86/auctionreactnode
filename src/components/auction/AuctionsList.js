import React from 'react';
import AuctionSummary from './AuctionSummary';

const AucitonsList = ({ auctions }) => {
  return (
    <div className="auctions-list section" >
      {auctions && auctions.map((auction, key) => {
          return (
            <AuctionSummary auction={auction} key={key} />
          );
        })}
    </div>
  );
}

export default AucitonsList;