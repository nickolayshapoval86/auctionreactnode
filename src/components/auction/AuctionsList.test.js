import { shallow } from 'enzyme';
import React from 'react';
import AuctionsList from './AuctionsList';

test('AuctionsList snapshot', () => {
    const mockAuctions = [{
        title:'Test auction title',
        description: 'Auction description text',
        creator: {
            name: 'Auction creator'
        }
    }];
    expect(shallow(<AuctionsList auctions={mockAuctions} />)).toMatchSnapshot();
});