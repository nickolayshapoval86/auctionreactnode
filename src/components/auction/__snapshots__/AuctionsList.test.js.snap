// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuctionsList snapshot 1`] = `ShallowWrapper {}`;
